backend/database/migrations/20190819164240-create-products.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
       type: Sequelize.STRING,
     },
     votes: {
```

Comment:
this votes field should also vanish from the products tables. since we have already created its own table.